FWCore/Integration/test/unit_test_outputs/testSubProcess.grep2.txt

```diff
@@ -147,7 +148,11 @@
 ++++ finished: begin job for module with label 'path3' id = 27
 ++++ starting: begin job for module with label 'path4' id = 28
 ++++ finished: begin job for module with label 'path4' id = 28
-++ finished: begin job
+++ starting: begin stream 0
+++ finished: begin stream 0
+++ starting: begin stream 0
+++ finished: begin stream 0
+++ starting: begin stream 0
```

Comment:
Guessing from the file name this tests definitively has SubProcess(es). Just to think out loud beyond this PR, would it be feasible to have the subprocess name as part of the printout from Tracer? (OTOH, SubProcesses are not used in practice, so I'm not sure how much effort is worth to put here)

Reply:
The information is already available in the Context objects. At the top of the chain the ProcessContext knows its process name and has a pointer to the parent ProcessContext if it is a SubProcess. I think this goes beyond the scope of this PR though. It is work in the Tracer to print this info out. And I agree that given that nobody is using SubProcess it is possibly a waste of time...

Reply:
Thanks. I fully agree improving the printout would be out of scope of this PR. Something to be thought about if/when we consider the fate of SubProcesses in general.